lib/storefront/checkmark-2.svg

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="16px" height="16px" viewBox="0 0 16 16" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <title>Checkmark variant 2</title>
+    <desc>A "verified" checkmark</desc>
+    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="verified" fill-rule="nonzero">
+            <g id="verified-check" fill="#6BA5D5">
```

Comment:
if we want to apply the color with CSS, then this fill should be removed

Reply:
as of now, we don't. if that changes we can modify in the future